styles/styles.css

```
@@ -82,19 +124,42 @@ body {
     font-weight: 700;
     text-transform: uppercase;
 }
+.advantages-pic{
+    Width: 800px;
+    margin: 0;
+    padding: 0;
+}
+.advantages-pic:nth-child(odd){
+    margin: 0;
+    margin-left: auto;
+}
+.advantages-item{
+    Width: 400px;
+}
 .advantages-item:nth-child(odd){
     background-color: rgba(131, 179, 211, 0.12);
 }
 .advantages-item:nth-child(even){
     background-color: rgba(131, 179, 211, 0.2);
 }
+.advantages-sub-list{
+    display: flex;
+    margin: 0;
+    padding: 0;
+}
+.advantages-text-wrapper{
+    width: 400px;
+}
 .advantages-item.pic{
+    display: flex;
     color:#FFFFFF;
     background-color: rgba(131, 179, 211);
 }
 .advantages-sub-item{
+    Width: 400px;
```

```suggestion
    width: 400px;
```
Заглавная буква в названии свойства.

---

Не заметил изменений.